packages/action-bar/src/uxp-action-bar.css

```diff
@@ -11,3 +11,77 @@ governing permissions and limitations under the License.
 */
 
 /* write uxp style overrides */
+
+:host {
+    height: 0;
+    bottom: 0;
+    top: 0;
+    margin-top: var(--mod-closebutton-margin-top);
+    margin-left: var(--mod-closebutton-margin-inline);
+    margin-right: var(--mod-closebutton-margin-inline);
+}
+
+:host([open]) {
+    height: calc(
+        var(
+                --mod-actionbar-spacing-outer-edge,
+                var(--spectrum-actionbar-spacing-outer-edge)
+            ) + var(--mod-actionbar-height, var(--spectrum-actionbar-height))
+    );
+}
+
+#popover {
+    height: var(--mod-actionbar-height, var(--spectrum-actionbar-height));
+    width: 100%;
+}
+
+.close-button {
+    margin-top: calc(
+        var(
+                --mod-actionbar-spacing-close-button-top,
+                var(--spectrum-actionbar-spacing-close-button-top)
+            ) / 2
+    );
+    margin-right: var(
+        --mod-actionbar-spacing-close-button-end,
+        var(--spectrum-actionbar-spacing-close-button-end)
+    );
+    margin-left: var(
+        --mod-actionbar-spacing-close-button-start,
+        var(--spectrum-actionbar-spacing-close-button-start)
+    );
+}
+
+.field-label {
+    margin-top: calc(
+        var(
+                --mod-actionbar-spacing-item-counter-top,
+                var(--spectrum-actionbar-spacing-item-counter-top)
+            ) / 1.5
+    );
+    margin-right: var(
+        --mod-actionbar-spacing-item-counter-end,
+        var(--spectrum-actionbar-spacing-item-counter-end)
+    );
+}
+
+.action-group {
+    margin-top: calc(
+        var(
+                --mod-actionbar-spacing-action-group-top,
+                var(--spectrum-actionbar-spacing-action-group-top)
+            ) / 2
+    );
+    margin-right: var(
+        --mod-actionbar-spacing-action-group-end,
+        var(--spectrum-actionbar-spacing-action-group-end)
+    );
+    margin-left: auto;
+}
+
+.action-group ::slotted([slot='buttons']:not(:last-child)) {
+    margin-right: var(
+        --mod-actiongroup-horizontal-spacing-regular,
+        var(--spectrum-actiongroup-horizontal-spacing-regular)
+    );
+}
```

Comment:
are focus related styles working for action bar?

Reply:
action-bar itself doesn't has any focus styles rather contained elements like clear-button and action button receive the focus when acted upon. Currently issue with buttons is their that focus ring is not being drawn properly that exist inside it as well. That we are going to handle separately when focus issue of button is picked.